lib/workers/repository/cache.ts

```diff
@@ -7,11 +7,8 @@ import type {
   BranchCache,
   BranchUpgradeCache,
 } from '../../util/cache/repository/types';
-import {
-  getBranchCommit,
-  getBranchParentSha,
-  isBranchModified,
-} from '../../util/git';
+import { getBranchCommit, isBranchModified } from '../../util/git';
+import { getCachedBranchParentShaResult } from '../../util/git/parent-sha-cache';
```

Comment:
why not re-export from git, so we only have one import?

Reply:
There are more like this so okay to handle in separate PR, thoughts? :)